tests/expectations/core/conftest.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pandas as pd
+import pytest
+
+from great_expectations.data_context.util import file_relative_path
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="module")
```
Should we change the scope of this fixture to prevent repeated reads of the CSV file and speed up the tests?
https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

The above assumes this won't lead to any test-leakage which may be incorrect.

Reply:
This is a great suggestion @Kilo59 thank you :)